googlemaps/client.py

```diff
@@ -71,12 +79,12 @@ def __init__(self, key=None, client_id=None, client_secret=None,
             seconds.
         :type retry_timeout: int
 
-        :param client_id: (for Maps API for Work customers) Your client ID.
-        :type client_id: string
-
-        :param client_secret: (for Maps API for Work customers) Your client
-            secret (base64 encoded).
-        :type client_secret: string
+        :param queries_per_second: Manually specify the number of queries
```

Comment:
Actually, can we rephrase this a bit? It's not super clear what happens if you don't specify this param. I'm also not sure you need to talk about sensible defaults.

Also, is it worth actually having 10 as the _actual_ default - as most devs won't ever set this? @markmcd